packages/cozy-procedures/src/locales/fr.json

```diff
@@ -1,71 +1,95 @@
 {
-  "personalData": {
-    "title": "Informations personnelles",
-    "subtitle": "Complétez les informations suivantes pour finaliser votre dossier"
+  "confirm": "Valider",
+  "overview": {
+    "subtitle": "Complétez les informations suivantes pour demander un crédit",
+    "request": "Crédit demandé",
+    "amount": "Montant",
+    "duration": "durée",
```

Comment:
really lowercase ?

Reply:
Yeah, same with `duration`. In the UI it forms a sentence `Amount over duration`. Then again should the casing really be done in a translation file..?